scipy/optimize/tests/test__differential_evolution.py

```diff
@@ -756,6 +757,9 @@ def c1(x):
                                      constraints=constraints)
 
         f_opt = 680.6300599487869
+        x_opt = (2.330499, 1.951372, -0.4775414, 4.365726, -0.6244870, 1.038131, 1.594227)
```

Comment:
I think the reason I didn't include this `x_opt` before was that there was too much variability in the solution to make an `assert_allclose(res.x, x_opt)` meaningful (the tolerances would have to be too big). But hopefully we can improve the accuracy, so it's good to add this.